### PR TITLE
Task v3 + pop v5 + mods + github actions

### PR DESCRIPTION
**Pull request by paganotoni**

### CODE REVIEW

**Comment by paganotoni** [comment]
Thanks @mclark4386! Maybe @stanislas-m or @markbates can take a look. I know @stanislas-m worked on the fixture implementation.

**Comment by stanislas-m** [comment]
The code seems OK, but I want to test it a bit before giving the approval.
Also we need to complete the comments, `// JSON ...`-like comments are not OK. :)

**Comment by paganotoni** [comment]
Great. I'll go for those comments soon. Thanks @stanislas-m

File: go.mod [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+require (
+	github.com/BurntSushi/toml v0.3.1
+	github.com/gobuffalo/buffalo v0.15.5
+	github.com/gobuffalo/envy v1.9.0
+	github.com/gobuffalo/httptest v1.5.0
+	github.com/gobuffalo/mw-csrf v1.0.0
+	github.com/gobuffalo/packd v1.0.0
+	github.com/gobuffalo/packr/v2 v2.8.0
+	github.com/gobuffalo/plush v3.8.3+incompatible
```
Comment (markbates) [inline go.mod]:
```suggestion
	github.com/gobuffalo/plush/v4
```

File: go.mod [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+	github.com/gobuffalo/pop/v5 v5.0.9
+	github.com/gofrs/uuid v3.2.0+incompatible
```
Comment (markbates) [inline go.mod]:
```suggestion
	github.com/gofrs/uuid/v3
```

**Comment by paganotoni** [comment]
@markbates @stanislas-m added comments and updated dependencies.

**Pull request merged**